ui-tests/src/test/java/org/openmrs/reference/page/RegistrationEditSectionPage.java

```diff
@@ -116,10 +117,13 @@ public void enterPhoneNumber(String phone) {
     public String getInvalidPhoneNumberNotification() {
         List<String> errors = getValidationErrors();
         String search = "valid phone number";
+        if (errors != null) {
         for (String str : errors) {
             if (str.trim().contains(search))
                 return str;
         }
+        return str;
```

Comment:
Few things here:

* Adding an `if` means the stuff inside the block should be indented a further level
* `if` statements should always use curly brackets even if the brackets are unnecessary (see [here](https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-CodeStyle)).
* The last `return str;` line doesn't seems to have the variable `str` in scope AFAICT. This is because `str` is only declared inside the for loop, so we can just eliminate it and fallback to returning `null`.

```suggestion
	          for (String str : errors) {
	              if (str.trim().contains(search)) {
	                  return str;
	              }
	          }
```

Reply:
Thanks @ibacher  ,Let me collect that